docs/networkpolicy-recommendation.md

```diff
@@ -0,0 +1,143 @@
+- [Perform NetworkPolicy Recommendation](#perform-networkpolicy-recommendation)
+  - [Run a policy recommendation job](#run-a-policy-recommendation-job)
+  - [Check the status of a policy recommendation job](#check-the-status-of-a-policy-recommendation-job)
+  - [Retrieve the result of a policy recommendation job](#retrieve-the-result-of-a-policy-recommendation-job)
+<!-- /toc -->
+
+## Introduction
+
+Theia NetworkPolicy Recommendation recommends the NetworkPolicy configuration
+to secure Kubernetes network and applications. It analyzes the network flows
+collected by [Grafana Flow Collector](
+network-flow-visibility.md#grafana-flow-collector) to generate
+[Kubernetes NetworkPolicies](
+https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+or [Antrea NetworkPolicies](
+https://github.com/antrea-io/antrea/blob/main/docs/antrea-network-policy.md).
+This feature assists cluster administrators and app developers in securing
+their applications according to Zero Trust principles.
+
+## Prerequisite
+
+- [Flow Exporter](
+https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#flow-exporter)
+is enabled
+- [Flow Aggregator](
+https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#flow-aggregator)
+is enabled and stores Pod labels
+- [Theia](
+https://github.com/antrea-io/theia/blob/main/docs/network-flow-visibility.md)
+is deployed and Spark Operator is enabled
+
+## Perform NetworkPolicy Recommendation
+
+Users can leverage Theia's NetworkPolicy Recommendation feature through `theia`
+CLI. `theia` is the command-line tool which provides access to Theia network
+flow visibility capabilities. To get more information about `theia`, please
+refer to this [doc](theia.md).
+
+There are 3 `theia` commands for the NetworkPolicy Recommendation feature:
+
+- `theia policy-recommendation run`
+- `theia policy-recommendation status`
+- `theia policy-recommendation retrieve`
+
+Or you could use `pr` as a short alias of `policy-recommendation`:
+
+- `theia pr run`
+- `theia pr status`
+- `theia pr retrieve`
+
+To see all options and usage examples of these commands, you may run
+`theia policy-recommendation [subcommand] --help`.
+
+### Run a policy recommendation job
+
+The `theia policy-recommendation run` command triggers a new policy
+recommendation job.
+If a new policy recommendation job is created successfully, the
+`recommendation ID` of this job will be returned:
+
+```bash
+$ theia policy-recommendation run
+Successfully created policy recommendation job with ID e998433e-accb-4888-9fc8-06563f073e86
+```
+
+`recommendation ID` is a universally unique identifier ([UUID](
+https://en.wikipedia.org/wiki/Universally_unique_identifier)) that is
+automatically generated when creating a new policy recommendation job. We use
+`recommendation ID` to identify different policy recommendation jobs.
+
+A policy recommendation job may take a few minutes to an hour to complete
```

Comment:
Seems "an hour" is an accurate timing, not sure if we should use "more than an hour".